Add social media validations to mentee application model for mentee application form

elhalvers opened the pull request:
## What's the change?
Added social media validations to user_mentee_application model for mentee application form
## What key workflows are impacted?
mentee application form

## Demo / Screenshots
https://github.com/agency-of-learning/PairApp/assets/74928397/4cbe89ab-7da2-48ef-bf0a-06481e31ee92

## Issue ticket number and link
#314 

## Checklist before requesting a review
- Added specs for social media validations and also for the 'available_hours_per_week' validation
- Did not add a spec for:

`validates :city, :state, :country, :reason_for_applying, :learned_to_code, :project_experience,
    :available_hours_per_week, presence: true`

toyhammered commented:
Can we add a few specs for the new validations? You can see examples in the `pair_request_spec.rb` file.

elhalvers commented:
> Can we add a few specs for the new validations? You can see examples in the `pair_request_spec.rb` file.

Hey @toyhammered I added tests for the 2 social media validations that I added. I also added a spec for the 'available_hours_per_week' validation.

I did not add a spec for:
`validates :city, :state, :country, :reason_for_applying, :learned_to_code, :project_experience, :available_hours_per_week, presence: true`.

Please let me know if you would like me to add a spec for this validation.

Thanks! : )

toyhammered commented:
> > Can we add a few specs for the new validations? You can see examples in the `pair_request_spec.rb` file.
> 
> Hey @toyhammered I added tests for the 2 social media validations that I added. I also added a spec for the 'available_hours_per_week' validation.
> 
> I did not add a spec for: `validates :city, :state, :country, :reason_for_applying, :learned_to_code, :project_experience, :available_hours_per_week, presence: true`.
> 
> Please let me know if you would like me to add a spec for this validation.
> 
> Thanks! : )

This is perfect thank you! No need to add validations for presence checks.
